civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -70,3 +70,16 @@ $row-margin-bottom: 15px;
     }
   }
 }
+
+/*
+* This is a TEMP fix to fit the text of input controls like date picker within input box
+* It is hardcoded to 13px for now.
+*/
+.chr_leave-request-modal__input-font-size {
```

Comment:
this is just a hack, we don't need an entire new selector, can't you just reference classes/tags that are already there? this is a bit overkill